entity-services-e2e/build.gradle

```diff
@@ -3,12 +3,18 @@ apply plugin: 'application'
 
 mainClassName = "com.marklogic.entityservices.e2e.ExamplesLoader";
 
-dependencies {
-    compile('com.marklogic:data-movement:1.0-SNAPSHOT') {
-        exclude(group: 'com.google.guava')
-    }
-    compile('ch.qos.logback:logback-classic:1.1.7')
-    compile('com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.7.0')
+repositories { 
+  maven { 
+    url "http://developer.marklogic.com/maven2"
```

Comment:
That’s the only place where you can get EA builds. I don’t think we publish these to Maven Central.